lib/eslint.js

```diff
@@ -461,12 +461,18 @@ module.exports = (function() {
             });
         } catch (ex) {
 
+            // If the message includes a leading line number, strip it:
+            var message = ex.message;
+            var lineMarker = message.match(/^line \d+: */i);
```

Comment:
Is that a repeated space in the regex? I'd suggest leaving that off and then trim the result to make it clearer.